factory/config.go

```diff
@@ -61,10 +68,18 @@ func (c *Config) Validate() (err error) {
 		return fmt.Errorf("Configuration field missing")
 	}
 
+	if c.Configuration.Server == nil {
+		return fmt.Errorf("HttpServer field missing")
+	}
+
 	if len(c.Configuration.Gnbs) == 0 {
 		return fmt.Errorf("no gnbs configured")
 	}
 
+	if c.Configuration.Server.IpAddr == "POD_IP" {
```

Comment:
Can we place this block next after checking for Configuration.Server. Seems that these should be logically grouped.

Reply:
Now that the server check is removed, I believe we no more need to relocate this line